src/walletConnect/screens/useIsDappListed.ts

```diff
@@ -0,0 +1,16 @@
+import { useSelector } from 'react-redux'
+import { dappsListSelector } from 'src/dapps/selectors'
+
+export const useIsDappListed = (dappName?: string, dappUrl?: string) => {
```

Comment:
Would be nice to add a test suite for this to make sure it works the way we expect.

Reply:
done